### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by backwind1233**
As title.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Setup (please complete the following information):**
+ - OS: [e.g. iOS]
+ - IDE: [e.g. IntelliJ]
+ - Library/Libraries: [e.g. com.azure:azure-core:1.16.0 (groupId:artifactId:version)]
```
Comment (saragluna) [inline .github/ISSUE_TEMPLATE/bug_report.md:31]: Should we change this to some Spring Cloud Azure library?

File: .github/ISSUE_TEMPLATE/question-query-template.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+**Query/Question**
+A clear and concise ask/query.
+
+***Why is this not a Bug or a feature Request?***
+A clear explanation of why is this not a bug or a feature request?
+
+**Setup (please complete the following information if applicable):**
+ - OS: [e.g. iOS]
+ - IDE: [e.g. IntelliJ]
+ - Library/Libraries: [e.g. com.azure:azure-core:1.16.0 (groupId:artifactId:version)]
```
Comment (saragluna) [inline .github/ISSUE_TEMPLATE/question-query-template.md:19]: same here

**Review by saragluna — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/question-query-template.md (hunk 2 above)
Comment (backwind1233) [reply .github/ISSUE_TEMPLATE/question-query-template.md]: Updated

**Comment by backwind1233** [comment]
Closing this, redirect to https://github.com/Azure-Samples/azure-spring-boot-samples/pull/328/files

**Pull request closed without merging**